Rakefile

```diff
@@ -12,6 +12,9 @@ rescue LoadError => e
   warn "WARNING: #{e.message}"
 end
 
+task :metadata_lint do
```

Comment:
@jeefberkey This is where all of this type of information is housed and this information is identical to the meta-RPM so they should be tightly coupled.

The metadata_lint is in rake helpers but including what is above it enables FAR more rake tasks than we want up here.